intfsyncd/intfsync.cpp

```diff
@@ -47,6 +47,11 @@ void IntfSync::onMsg(int nlmsg_type, struct nl_object *obj)
         return;
 
     key = LinkCache::getInstance().ifindexToName(rtnl_addr_get_ifindex(addr));
+
+    /* Don't sync lo, eth0, and docker0 routes */
+    if (key == "lo" || key == "eth0" || key == "docker0")
+        return;
```

Comment:
for interface sync, we need to get lo address into the swss. This is becasue we are putting bgp loopback IP address on interface lo, and this one needs to be in the ASIC in order to trap all IP packets destined to bgp loopback address.

---
In reply to: [65829595](https://github.com/Azure/sonic-swss/pull/34#discussion_r65829595) [](ancestors = 65829595)